flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -44,13 +44,10 @@
 import java.util.List;
 import java.util.Random;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
 
 public class ExecutionConfigTest extends TestLogger {
```

Comment:
```suggestion
class ExecutionConfigTest {
```

`public` is no longer needed for test classes in JUnit5. 
It's also not needed for each `@Test`.
`TestLogger` has been reworked to be a Flink default test rule and is no longer needed.